dodo/dodo.spec

```diff
@@ -0,0 +1,31 @@
+Name:   dodo
+Version:    1.1
+Release:	1%{?dist}
+Summary:	RPM Package for dodo test runner
+
+License:    test_runner
```

Comment:
Please change license to None for now. We need to decide what license to use.

Reply:
Okay.